index.html

```diff
@@ -5,16 +5,158 @@
+
+                    "Have the courage to follow your heart and intuition. They somehow know what you truly want to
+                    become."
+
+                    "Let's go invent tomorrow rather than worrying about what happened yesterday."
+                    We don't get a chance to do that many things, and every one should be really excellent. Because
+                    this
+                    is our life. Life is brief, and then you die, you know? And we've all chosen to do this with our
+                    lives. So it better be damn good. It better be worth it."
+
+                    9. "Be a yardstick of quality. Some people aren't used to an environment where excellence is
+                    expected."
 
-    <!-- Add your markup here -->
+                    10. "You can't connect the dots looking forward; you can only connect them looking backward. So
+                    you
+                    have to trust that the dots will somehow connect in your future. You have to trust in
+                    something--your gut, destiny, life and karma, whatever. This approach has never let me down, and
+                    it
+                    has made all the difference in my life."
+                    11. "Here's to the crazy ones, the misfits, the rebels, the troublemakers, the round pegs in the
+                    square holes.... The ones who see things differently--they're not fond of rules.... You can
+                    quote
+                    them, disagree with them, glorify or vilify them, but the only thing you can't do is ignore them
+                    because they change things.... They push the human race forward, and while some may see them as
+                    the
+                    crazy ones, we see genius, because the ones who are crazy enough to think that they can change
+                    the
+                    world are the ones who do."
+                    <button class="button_about_me">
+                        Read More
+                    </button>
+                </div>
+                <div class="col-12 about_me_img">
+                    <img src="product.jpg" class="about_im_2">
+                </div>
+            </section>
+            <section>
+                <div class="section_three">
+                    How Can I help?
+                    <div class="help_font">
+                        <p>
+                            one on one coaching
+                            click here to add you own text and edit more.
+                        </p>
+                        <button class="button_section_three">
+                            Book it
+                        </button>
+                        <button class="fix">
+                            More services
+                        </button>
+                    </div>
+                </div>
+            </section>
+    </main>
+    <footer>
+        <div class="fo">
+        <div class="footer_style ">
```

Comment:
It would be good to set the background-color on the footer, and then on the next level down add a `container` - that your footer content is contained on the page, but the blue colour isn't.